### ISSUE TITLE
As a dev, I'd like to add SSO to my app using MSAL Angular so that users in Teams don't need to sign in one more time

### ISSUE THREAD

**Issue by waldekmastykarz** [sinedied/teams-angular#4]
Depends on #5

**Comment by waldekmastykarz** [comment]
Install the `@microsoft/teams-js` dependency. Add [TeamsGuard](https://github.com/waldekmastykarz/angular-teams-app/blob/main/src/app/teams.guard.ts) and [TeamsService](https://github.com/waldekmastykarz/angular-teams-app/blob/main/src/app/teams.service.ts). Extend AAD app with:

- [identifier URI](https://github.com/waldekmastykarz/angular-teams-app/blob/5502ebd8a548a7d7ce67cbe9316b0e22cb32b365/aad-app-manifest.json#L16) (the URI must be built as follows: `api://domain.from.tunnel/appId`
- [OAuth2 permission](https://github.com/waldekmastykarz/angular-teams-app/blob/5502ebd8a548a7d7ce67cbe9316b0e22cb32b365/aad-app-manifest.json#L32-L42)
- [reply URL pointing to the tunnel URL](https://github.com/waldekmastykarz/angular-teams-app/blob/5502ebd8a548a7d7ce67cbe9316b0e22cb32b365/aad-app-manifest.json#L32-L42)
- preauthorized applications (appIds are fixed values as shown in the snippet, permissionIds match the ID of the OAuth2 permission defined above):

   ```json
	"preAuthorizedApplications": [
		{
			"appId": "5e3ce6c0-2b1f-4285-8d4b-75ee78787346",
			"permissionIds": [
				"cf38eb5b-8fcd-4697-9bd5-d80b7f98dfc5"
			]
		},
		{
			"appId": "1fec8e78-bce4-4aaf-ab1b-5451cc387264",
			"permissionIds": [
				"cf38eb5b-8fcd-4697-9bd5-d80b7f98dfc5"
			]
		}
	],
   ```

**Comment by waldekmastykarz** [comment]
This is included in v0.1
